tools/flang2/flang2exe/iliutil.cpp

```diff
@@ -6134,7 +6134,7 @@ addarth(ILI *ilip)
 
   case IL_DASIN:
     if (XBIT_NEW_MATH_NAMES) {
-      fname = make_math(MTH_asin, &funcsptr, 1, false, DT_DBLE, 2, DT_DBLE,
+      fname = make_math(MTH_asin, &funcsptr, 1, false, DT_DBLE, 1, DT_DBLE,
```

Comment:
You should only specify one argument data type instead of two:
```c
      fname = make_math(MTH_asin, &funcsptr, 1, false, DT_DBLE, 1, DT_DBLE);
```

Reply:
Thanks for noticing this.